.gitignore

```diff
@@ -8,6 +8,7 @@
 /.bundle
 /Gemfile.lock
 /*.un~
+/.bye*
```

Comment:
Fixed up my global ignore and rewrote history removing Gemfile.lock (I have no idea why I added this ever) and removing *.un~ as well.